### PR TITLE
2054 Create PlayScreen

### PR DESCRIPTION
**Pull request by HoangLongBFramgia**
# SUMMARY
- Create UI PlayScreen
- Create PlayActivity
- Insert Image 
# Redmine 
- https://edu-redmine.framgia.vn/issues/2054
# BUGLIST

### CODE REVIEW

**Comment by HoangLongBFramgia** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -24,6 +23,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".screen.play.PlayActivity"></activity>
     </application>
 
 </manifest>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: add end of file

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -24,6 +23,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".screen.play.PlayActivity"></activity>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: self close tag

**Comment by DoanVanToan** [comment]
commented
1 chút comment nhỏ nhắn

**Comment by HoangLongBFramgia** [comment]
ready

**Pull request merged**